libs/dkg/src/dkg_service.cpp

```diff
@@ -433,6 +450,7 @@ State DkgService::OnCollectSignaturesState()
     }
   }
 
+  // TODO(HUT): looks like a bug here
```

Comment:
@ejfitzgerald chat in the office? Looks like if round->HasSignature() gets set it can never get out of this state